pkg/segment/utils/timeconsts.go

```diff
@@ -0,0 +1,35 @@
+package utils
+
+import "fmt"
+
+type TimeUnit uint8
+
+const (
+	TMUs TimeUnit = iota
+	TMMs
+	TMCs
+	TMDs
```

Comment:
Can ypu spell put what those those  Ms, Cs,Ds mean just kike you fid for second minute etc